engine/src/main/java/org/hibernate/search/engine/search/predicate/spi/SimpleQueryStringPredicateBuilder.java

```diff
@@ -20,7 +22,15 @@
 
 	void skipAnalysis();
 
-	interface FieldState {
+	void enable(SimpleQueryFlag operation);
+
+	void enable(EnumSet<SimpleQueryFlag> operations);
+
+	void disable(SimpleQueryFlag operation);
+
+	void disable(EnumSet<SimpleQueryFlag> operations);
+
+	interface FieldState<B> {
```

Comment:
Why the `<B>`?

Reply:
I'm withdrawing it. I don't remember why, but I don't need it.